src/main/java/pl/edu/agh/firecell/model/ModelChecker.java

```diff
@@ -0,0 +1,32 @@
+package pl.edu.agh.firecell.model;
+
+import pl.edu.agh.firecell.model.exception.ConductionCoefficientException;
+import pl.edu.agh.firecell.model.material.ConductionCoefficientProvider;
+import pl.edu.agh.firecell.model.material.Material;
+
+public class ModelChecker {
+
+    private void checkConductionCoefficientProvider() throws ConductionCoefficientException{
```

Comment:
Move this function to the `ModelCondeuctionMap`, since it's only connected with it.
Rename for `validate()` and run as the last line of a constructor, after creation of an actual map.
What is the point of creating a class, that could be not working, and counting on somebody else to check that?